.github/workflows/clj-vuln-scan.yml

```diff
@@ -0,0 +1,90 @@
+name: Clojure Vulnerability Scan
+on:
+  workflow_call:
+    inputs:
+      runner:
+        required: false
+        type: string
+        default: self-hosted
+      target_branch:
+        required: false
+        type: string
+        default: main
+      specified_ref:
+        required: false
+        type: string
+      docker_run_envs:
+        required: false
+        type: string
+        description: Optional environment variables to pass to docker run command (formatted '--env <VAR_NAME>=<var_value> --env <VAR_NAME>=<var_value> ...')
+jobs:
+  scan:
+    name: Scan Dependencies
+    runs-on: ${{ inputs.runner }}
+    concurrency:
+      group: ${{ github.workflow }}-${{ github.event.inputs.specified_ref || github.ref }}
```

Comment:
📓 I believe this will work correctly, but won't be able to test until rentpath/rp-healthchecker-clj#13 is merged